src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -33,13 +39,17 @@ public ResponseEntity<Object> getAllIssuers() {
     @GetMapping("/{issuer-id}")
     public ResponseEntity<Object> getIssuerConfig(@PathVariable("issuer-id") String issuerId) {
         ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
-        //TODO: Modify id
-        responseWrapper.setId("mosip.inji.properties");
+        responseWrapper.setId(ID);
         responseWrapper.setVersion("v1");
         responseWrapper.setResponsetime(DateUtils.getRequestTimeString());
 
 
-        responseWrapper.setResponse(issuersService.getIssuerConfig(issuerId));
+        IssuerDTO issuerConfig = issuersService.getIssuerConfig(issuerId);
+        responseWrapper.setResponse(issuerConfig);
```

Comment:
Why are we setting response here?
If it's an error, then there should not be response field available

Reply:
Refactored

Reply:
I was saying, error response should not have field for 2xx response.
so here error response should be like with other attributes like status:404 if needed
```
{
    "errorCode":"code",
    "errorMessage":"message"
}
```